app/views/pages/accessibility_statement.html.erb

```diff
@@ -87,65 +84,64 @@
       <h2 class="govuk-heading-m">Enforcement procedure</h2>
 
       <p>
-        The <%= link_to 'Equality and Human Rights Commission (EHRC)', 'https://www.equalityhumanrights.com/en' %>
-        is responsible for
-        enforcing the 'accessibility regulations' - officially known as the
-        <%= link_to 'Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.',
-          'http://www.legislation.gov.uk/uksi/2018/852/contents/made' %>
+        The Equality and Human Rights Commission (EHRC) is responsible for enforcing the
+        Public Sector Bodies (Websites and Mobile Applications)
+        (No. 2) Accessibility Regulations 2018 (the 'accessibility regulations').
       </p>
 
       <p>
         If you're not happy with how we respond to your complaint, contact the
-        <%= link_to 'Equality Advisory and Support Service (EASS)', 'https://www.equalityadvisoryservice.com/' %>.
+        <%= link_to 'Equality Advisory and Support Service (opens in a new tab)', 'https://www.equalityadvisoryservice.com/', target: "_blank" %>.
       </p>
     </section>
 
     <section>
-      <h2 class="govuk-heading-m">Technical information about this service's accessibility</h2>
+      <h2 class="govuk-heading-m">Technical information about this website's accessibility</h2>
 
       <p>
-        DfE is committed to making its service accessible, in accordance with the
+        The Department for Education is committed to making its websites accessible, in accordance with the
         Public Sector Bodies (Websites and Mobile Applications) (No. 2)
         Accessibility Regulations 2018.
       </p>
+    </section>
 
+    <section>
+      <h2 class="govuk-heading-m">Compliance status</h2>
       <p>
-        This service is fully compliant with the
-        <%= link_to 'Web Content Accessibility Guidelines version 2.1', 'https://www.w3.org/TR/WCAG21/' %>
-        AA standard.
+        This service is partially compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard, due to the following non-compliances:
       </p>
-
-      <h3 class="govuk-heading-s">
-        Navigation and accessing information
-      </h3>
-
       <p>
-        We have not identified any problems relating to interactive tools and
-        transactions.
+         <ul class="govuk-list govuk-list--bullet">
+        <li>if a candidate tries to search for a school experience placement without entering a postcode,
+          we do not surface an error message.
+          This does not meet <%= link_to 'WCAG version 2.1 A – 3.3.1 Error Identification (opens in a new tab)', 'https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html' %>
+          and <%= link_to 'AA – 3.3.3 Error Suggestion', 'https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html', target: "_blank" %>.</li>
```

Comment:
```suggestion
          and <%= link_to 'WCAG version 2.1 AA – 3.3.3 Error Suggestion', 'https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html', target: "_blank" %>.</li>
```
I would add this in again to be really clear - especially given that WCAG 2.2 is around the corner